iotdb/src/main/java/org/apache/iotdb/db/service/TSServiceImpl.java

```diff
@@ -191,29 +193,30 @@ public TSCloseOperationResp closeOperation(TSCloseOperationReq req) throws TExce
       releaseQueryResource(req);
 
       clearAllStatusForCurrentRequest();
-    } catch (FileNodeManagerException e) {
+    } catch (Exception e) {
       LOGGER.error("Error in closeOperation : {}", e.getMessage());
     }
     return new TSCloseOperationResp(new TS_Status(TS_StatusCode.SUCCESS_STATUS));
   }
 
-  private void releaseQueryResource(TSCloseOperationReq req) throws FileNodeManagerException {
+  public void releaseQueryResource(TSCloseOperationReq req) throws Exception {
```

Comment:
Because in cluster module, it will throws other kind of Excepetion which is defined in cluster nodule. What's more, the method `closeOperation()` which call `releaseQueryResource ()` is `try{releaseQueryResource()} catch (Exception e)`. So it's ok to change the throws Exception.